packages/api/test/addbuild-getbuild.js

```diff
@@ -341,8 +351,7 @@ describe('Add-Build', () => {
 
       assert(ansObj[2].name === 'a/4'); // failed on build before the last build
 
-      assert(ansObj[3].name === 'a/3' || ansObj[3].name === 'a/1'); // both failed on most recent build
```

Comment:
With the new implementation, not every test will be updated in the queue.  As in this case, test `a/1` was considered to be a consistently healthy test, so it will no longer show up